test/test_runner.py

```diff
@@ -74,8 +74,6 @@ def test_run_benchmarks(benchmarks_fixture, tmpdir):
         append_samples=True, record_samples=True)
     times = ResultsWrapper(results, b)
 
-    end_timestamp = datetime.datetime.utcnow()
```

Comment:
Could you please just erase the name of the variable and leave the rest of the code, something like:
`datetime.datetime.utcnow()`